src/printer-graphql.js

```diff
@@ -22,7 +22,10 @@ function genericPrint(path, options, print) {
 
   switch (n.kind) {
     case "Document": {
-      return join(concat([hardline, hardline]), path.map(print, "definitions"));
+      return concat([
+        join(concat([hardline, hardline]), path.map(print, "definitions")),
```

Comment:
We'll probably want to respect original newlines instead of forcing one empty line, but we can deal with it later.